### ISSUE TITLE
Proton 7.0-6 and newer doesn't properly initialize window with fractional scaling

### ISSUE THREAD

**Issue by jonnyrobbie** [ValveSoftware/Proton#6785]
When trying to run games with proton 7.0-6 and newer (I've tried 7.0-6, 8.0-2, Experimental and GE), the game tries to launch, the sounds/music starts to be playing in the background, but the window surface keeps showing desktop/steam.

It almost seems like the game "works" underneath all of this (hard maybe) as I don't see the mouse and I cannot interact with the desktop until I press the keyboard shortcut for minimizing window. But until then, the window surface doesn't get drawn into, the buffer just keeps the last image it has seen, that being my desktop and steam after clicking the Play button.

Proton version 6.3-8 and 5.13-6  seems to work fine, so this looks like some kind of regression of the way proton handles wayland fractional scaling. When I set scaling to 100% it seems to work fine.

Attached is a proton log for Doom 2016, but it's not a game specific issue as the problem is also with Planet Zoo and other games too. steam-native vs steam-runtime also doesn't make a difference.
The log gets chock full of hundreds of thousands of assembly stack traces mere seconds after launch, so I've had to trim that down to first 100 lines.

Arch Linux (latest), KDE on wayland, AMD/AMD, kwin fractional scaling set to 150%. Doesn't work either when I set scaling for legacy apps to be scaled by the system nor to let the app scale by themselves.

```
$ uname -roms 
Linux 6.3.1-arch2-1 x86_64 GNU/Linux
$ kwin_wayland -v
kwin 5.27.5
```

**Comment by sggutier** [comment]
I used to have this issue as well. As a workaround (?) I enabled the "Scaled by the system" option in Display Configuration, and now Proton launches without issues on 7.0-6+ without issues:

![image](https://github.com/ValveSoftware/Proton/assets/5473302/570d2310-1d95-478b-af20-c402467ec13e)

**Comment by jonnyrobbie** [comment]
Yes, but as of right now, it is a workaround with all the disadvandates this option brings in. Mainly, steam client doesn't support wayland fractional scaling, so it's window is unreasonably small, etc.

**Comment by yswtrue** [comment]
Same here

**Comment by yswtrue** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/6785#issuecomment-1575881936

this works for me

**Comment by yswtrue** [comment]
open wincfg by protontricks
and disable `window manager to decorate window` and `window manager to control window` like this and save and it works fine now.
![image](https://github.com/ValveSoftware/Proton/assets/1270490/ab50ab76-b155-4922-8bfd-46aec162ba17)
